grr/server/grr_response_server/gui/validate_iap.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Sample showing how to validate the Identity-Aware Proxy (IAP) JWT.
+
+This code should be used by applications in Google Compute Engine-based
+environments (such as Google App Engine flexible environment, Google
+Compute Engine, or Google Container Engine) to provide an extra layer
+of assurance that a request was authorized by IAP.
+
+For applications running in the App Engine standard environment, use
+App Engine's Users API instead.
+"""
```

Comment:
I have just a minor request: we are trying to make our codebase compatible with Python 3. One of the things we do to ensure that is adding [`__future__` imports](https://docs.python.org/2/library/__future__.html). In all new files we try to at have for `absolute_import`, `division` and `unicode_literals` (for example, see [this module](https://github.com/google/grr/blob/8c94f4f836b3944c7cebc08a8416cbd0c2a0ad21/grr/core/grr_response_core/lib/util/compatibility.py#L3)).

This of course does not of fully enforce Python 3 compatiblity, but at least adds some level of convidence and brings us one step closer to our the goal.

Reply:
👍